### ISSUE TITLE
Relationship of SWEET terms to prior versions

### ISSUE THREAD

**Issue by dr-shorthair** [ESIPFed/sweet#21]
Each version of SWEET uses a new namespace/URI stem. 
These include the version number, so the URI for terms with the same local name is different between versions. This means that strictly no relationship can be inferred between the same term used in different versions. 

If the current naming convention (including version number) persists, then there should be explicit links back to the equivalent concept in prior versions of SWEET (maybe use dct:replaces as the predicate).

**Comment by lewismc** [comment]
I think it is the general consensus now that we will NOT be using the current naming convention (including version number) for representing URI's. The community is currently moving towards adopting a new [URI transition and governance](https://github.com/ESIPFed/sweet/issues/6) process.

**Comment by lewismc** [comment]
@dr-shorthair is this now resolved due to th new URI structure and use of ```owl:versionInfo "2.4" ;```. I suggest for each new version we increment the appropriate version.

**Comment by dr-shorthair** [comment]
Probably. 

However, why "2.4"? 
dotty versions are so last millenium. 
Is "2.4" intended to convey the idea that this is only incremental change? 
If so, then that's probably wrong. If not, then why not take this opportunity to just go to 
   owl:versionInfo "3" ;
and then the next version can be "4" and the one after that "5" and so-on. 

Too much weight on fancy version numbering systems can get you all tangled up, and maybe also constipated.

**Comment by carueda** [comment]
Although more targeted to software (and very widely used btw), http://semver.org/ is a relevant reference, at least conceptually. In concrete, upon new versions of SWEET, do we want to reflect in the version label things like: 
- new version brings incompatible changes;
- new version has additional definitions but overall the new version is backwards-compatible
- new version only with some fixes (but overall, still backwards-compatible)

**Comment by lewismc** [comment]
@dr-shorthair 

> Is "2.4" intended to convey the idea that this is only incremental change?

No, I introduced it merely to convey that fact that there was a change... that is all. Versioning has never been discussed. We are doing that now!

> If so, then that's probably wrong. If not, then don't send mixed messages.

AFAIK no one is advocating sending mixed messages. We can clarify this now.

> Why not take this opportunity to just go to owl:versionInfo "3" ; and then the next version can be "4" and the one after that "5" and so-on.

ACK, I agree. PR coming now.

> Too much weight on fancy version numbering systems can get you all tangled up, and maybe also constipated.

There is not much weight. All we need to do is build consensus. I think we've found it.

@carueda 

> Although more targeted to software (and very widely used btw)...

To be honest I agree. I don't see much of an issue with semantic versioning. We are after all dealing with a software artifact, or at least an artifact which is consumed by software. As long as we make clear, concise and helpful announcements alongside future SWEET releases I see absolutely no reason why we cannot maintain an ```owl:versionInfo``` which follows semantic versioning. 

I do think that we should skip to ```owl:versionInfo "3" ;``` for the next release though. At the time I created 2.4 I did not entirely anticipate URI changes, etc.
